mvpa2/tests/test_lib_afni.py

```diff
@@ -0,0 +1,57 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the PyMVPA package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Unit tests for PyMVPA atlases"""
+
+import numpy as np
+import nibabel as nb
+from mvpa2.testing import *
+
+skip_if_no_external('nibabel')
+
+from mvpa2.support.afni import lib_prep_afni_surf
+
+@sweepargs(mock_3dinfo=[False,True])
+@sweepargs(is_plumb=[False,True])
+@with_tempfile('.nii', 'test_plump')
+def test__ensure_expvol_is_plump(filename, is_plumb, mock_3dinfo):
+    if not mock_3dinfo:
+        # AFNI's 3dinfo is required
+        skip_if_no_external('afni-3dinfo')
```

Comment:
overall -- neat.  Just don't forget to define a test for `afni-3dinfo` within  `mvpa2/base/externals.py` or let me know if you need help with that